universitat-bremen-institut-fur-politikwissenschaft.csl

```diff
@@ -67,24 +67,40 @@
   </macro>
   <macro name="access">
     <choose>
-      <if type="webpage article-journal article-magazine article-newspaper article speech" variable="URL" match="all">
-        <group>
-          <text value="Text abrufbar unter:" prefix=" " suffix=" "/>
-          <text variable="URL"/>
-          <group prefix=" (" suffix=")">
-            <text value="Zugriff am" suffix=" "/>
-            <date variable="accessed">
-              <date-part name="day" form="numeric" suffix="."/>
-              <date-part name="month" form="numeric" suffix="."/>
-              <date-part name="year" form="long"/>
-            </date>
-          </group>
-        </group>
+      <if type="webpage article-journal article-magazine article-newspaper article entry-encyclopedia speech post-weblog interview" variable="URL" match="all">
```

Comment:
I'm not sure why we're testing for item type here at all? These would seem to be general rules for all item types, no?